tests/attr/test_occlusion.py

```diff
@@ -15,9 +18,9 @@
 
 
 class Test(BaseTest):
-    def test_improper_window_shape(self):
+    def test_improper_window_shape(self) -> None:
         net = BasicModel_ConvNet_One_Conv()
-        inp = torch.arange(16).view(1, 1, 4, 4).type(torch.FloatTensor)
+        inp = torch.arange(16).view(1, 1, 4, 4).to(torch.float)
```

Comment:
nit: Can you actually update this (and below) to `torch.arange(16, dtype=torch.float).view(1, 1, 4, 4)`? That would be a little more concise.